Sources/Sentry/Public/SentryHub.h

```diff
@@ -2,7 +2,7 @@
 #import "SentryIntegrationProtocol.h"
 
 @class SentryEvent, SentryClient, SentryScope, SentrySession, SentryUser, SentryBreadcrumb,
-    SentryId, SentryUserFeedback, SentryEnvelope;
+    SentryId, SentryUserFeedback, SentryEnvelope, SentryTransactionContext, SentryTransaction;
```

Comment:
on Java/.NET, we call it only `TransactionContext`

Reply:
This was one of the first things I added, was trying to figure Objc SDK naming convention. I guess I can fallow Java naming correct?

Reply:
mmm usually we just call as it is, we only add `Sentry` in front of what could bring import conflicts, so we run into issues like https://github.com/getsentry/sentry-dart/issues/250

yeah I'd prefer to keep rather the same whenever possible.

Reply:
In Objective-C it's even more important to avoid conflicts since it'll happen even outside of an `import` like in Dart. So I'd say prefixing with Sentry here is probably the best option

Reply:
We tested it and it indeed conflicted. So if someone adds an OpenTelemetry library later it would not compile. This is why we have pretty much everything here prefixed with `Sentry`. I'll resolve this with this reason, feel free to reopen if strong disagree.

Reply:
that makes sense, actually, we had a bug with KSCrash about that, good catch